cucumber/api/features/retrieve_api_key.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Retrieving an API key with conjurctl
```

Comment:
I would add another test to verify we have error when the API is wrong(role does not exist).

Reply:
done